nixos/modules/system/boot/loader/raspberrypi/raspberrypi.nix

```diff
@@ -103,6 +103,6 @@ in
 
     system.build.installBootLoader = builder;
     system.boot.loader.id = "raspberrypi";
-    system.boot.loader.kernelFile = platform.kernelTarget;
```

Comment:
`linux-kernel` does not exist in this scope. `platform` comes from `inherit (pkgs.stdenv.hostPlatform) platform;` and is now unused.

This does not evaluate.

Reply:
Sorry, I will fix. How should I test this?

Reply:
Good questions. `nix-build -E '(import ./. {}).nixos { boot.loader.raspberryPi.enable = true; boot.loader.grub.enable = false; fileSystems."/".device = "e"; }'` should work.

Reply:
See https://github.com/NixOS/nixpkgs/pull/110803